public/app/features/teams/TeamGroupSync.tsx

```diff
@@ -90,14 +92,25 @@ export class TeamGroupSync extends PureComponent<Props, State> {
   render() {
     const { isAdding, newGroupId } = this.state;
     const { groups, isReadOnly } = this.props;
-
     return (
       <div>
+        {highlightTrial() && (
+          <UpgradeBox
+            featureId={'team-sync'}
+            eventVariant={'trial'}
+            featureName={'team sync'}
+            text={'Add a group to enable team sync for free during your 14-day trial of Grafana Pro.'}
```

Comment:
As mentioned on Slack, I don't think we should hard-code "14-day" in these texts.